misc/git-hooks/pre-commit

```diff
@@ -178,6 +178,13 @@ function check_typos() {
 }
 export -f check_typos
 
+function check_nix_fmt() {
```

Comment:
I think this file is auto-generated by flakebox, so the PR would have to be there (happy to do that for you if you want).

Also `git ls` output is available for re-use to avoid running it multiple times: `$FLAKEBOX_GIT_LS` or `$FLAKEBOX_GIT_LS_TEXT` (only text files).

Reply:
@elsirion more work was needed, done in https://github.com/rustshop/flakebox/pull/177